app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -28,12 +29,11 @@
 import timber.log.Timber;
 
 public class NewContestActivity extends BaseMvpActivity<NewContestPresenter> implements NewContestMvpContract.View, EditContestContract {
-    @BindView(R.id.toolbar)
-    Toolbar toolbar;
     @BindView(R.id.fragment_container)
     ViewPager viewPager;
     private SlidingTabAdapter tabAdapter;
     private ActionBar actionBar;
```

Comment:
np once we're on a shared history my activities will be inheriting this behavior